api/server/middleware/experimental.go

```diff
@@ -0,0 +1,29 @@
+package middleware
+
+import (
+	"net/http"
+
+	"golang.org/x/net/context"
+)
+
+// ExperimentalMiddleware is a the middleware in charge of adding the
+// 'Docker-Experimental' header to every outgoing request
+type ExperimentalMiddleware struct {
```

Comment:
ah, ok :angel: nevermind then :stuck_out_tongue_closed_eyes: We could (should) rename those I think, but it's not the scope of the PR 👼